[SRVCOM-1831] Serverless functions: update abstracts format (#46660)
openshift/openshift-docs: modules/serverless-nodejs-template.adoc

[id="serverless-nodejs-template_{context}"]
= Node.js function template structure

> When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` configuration file, used for specifying the image.

Reviewer: ```suggestion
When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` configuration file, which is used for specifying the image.
```

Author: @jeana-redhat Aren't we supposed to have two or more sentences in the first paragraph?

Author: Nvm, meant to comment another suggestion

> When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` configuration file, which is used for specifying the image.

Reviewer 2: ```suggestion
When you create a Node.js function using the `kn` CLI, the project directory looks like a typical Node.js project. The only exception is the additional `func.yaml` file, which is used to configure the function.
```
The func.yaml file is responsible for a lot more than simply specifying the image. It is responsible for the entire function configuration.
